ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionIndexingInfo.scala

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao.events
+
+import java.time.Instant
+
+import com.daml.ledger.{TransactionId, WorkflowId}
+import com.daml.ledger.participant.state.v1.{
+  CommittedTransaction,
+  DivulgedContract,
+  Offset,
+  SubmitterInfo
+}
+import com.daml.lf.ledger.EventId
+import com.daml.lf.transaction.BlindingInfo
+
+final case class TransactionIndexingInfo private (
```

Comment:
What does `private` change here? The `apply` is still public, no?

Reply:
It disallows creating an instance with the synthesized `apply` (but not with the one you implement yourself).

Reply:
"Disallows" outside of the private scope, of course.

Reply:
Does it?

Reply:
Ok, in case you override it, 👍

Reply:
It's called `from` here to make sure it's clear.

Going back to the original comment, as you mentioned offline my assumption about `apply` being `private` is wrong, so I'm removing it.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8019/commits/f3999e78642fbc7782c35c3a430737538eb13cee